RELEASE.md

```diff
@@ -1,5 +1,15 @@
 # Release notes
 
+## 1.1.4
+
+- Minumum Runtime Qt SDK version is 100.9.
+- Minimum Qt SDK version is Qt 5.15.0.
+- Minimum Android SDK version is now 21, target SDK version is 29.
+- Fixed erronious "missing file" dialog appearing on startup when opening an MSPK. This was a layer-to-json issue when saving the config.
+- Reverted the "ModelSceneSymbol to PictureMarkerSymbol" change. A camera bug was was introduced because a rotation-expression can not be used in place of a heading-expression when working with an `OrbitGeoElementCameraController`. We will need to park this upgrade until an enhancement is exposed by ArcGISRuntime. 
+- Fixed bug where layers were not written out to the config when the `PackageDirectory` key is missing.
+- Renamed "master" branch to "main."
```

Comment:
After thinking it over, I don't think we should include this line item in the RELEASE notes. Do you feel strongly about keeping this item here?

Reply:
If it were up to me I'd not have the word master anywhere in the repo. I'll remove this line.